modules/network/locals.tf

```diff
@@ -238,19 +246,35 @@ locals {
       port             = -1,
       stateless        = false
     },
+    {
+      description      = "Pod to Kubernetes API endpoint communication.",
+      destination      = local.cp_subnet,
+      destination_type = "CIDR_BLOCK",
+      protocol         = local.tcp_protocol,
+      port             = 6443,
+      stateless        = false
+    },
+    {
+      description      = "Pod to Kubernetes API endpoint communication.",
+      destination      = local.cp_subnet,
+      destination_type = "CIDR_BLOCK",
+      protocol         = local.tcp_protocol,
+      port             = 12250,
+      stateless        = false
+    }
   ]
 
   pods_ingress = [
     {
-      description = "Allow worker nodes to access pods.",
+      description = "Kubernetes API endpoint to pod communication.",
       protocol    = local.all_protocols,
       port        = -1,
```

Comment:
Can we be more specific re: protocols and ports?

Reply:
The rule here didn't change - the description for it vs. the one below seemed flipped based on source subnet. That said, I do think we can make them more specific, e.g. this one is documented but might not be necessary - I'll find out and follow up.

Reply:
Updating the description to reflect the use-case for API server calling webhooks served by pods.